src/util/hasValue.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Function that accepts data and determines whether that data has
+ * value or not
+ *
+ *
+ * @return boolean
+ * @param value
+ */
+
```

Comment:
docblocks should not have breaks after, it must be 'attached' to the function it describes